views/pages/details.html

```diff
@@ -43,6 +43,12 @@
   {{#record.status.courtOrderRevoked}}
     <div class="flag">{{#t}}flags.courtOrderRevoked{{/t}}</div>
   {{/record.status.courtOrderRevoked}}
+
+  {{#record.previousRegistration.systemNumber}}
+    <div class="flag warning">{{#t}}flags.previousRegistration{{/t}}
+      <a id="backToSearchResults" href="/details/{{record.previousRegistration.systemNumber}}?{{querystring}}" class="button">{{record.previousRegistration.date}}</a>
```

Comment:
I wonder if the whole flag should be a link. (We don't have anything like that at the moment.)

Alternatively it might be nicer if the word "original registration" was the link but of course that doesn't play nicely with the translation system.

Reply:
Oh and I think that "id" is wrong.

Reply:
Is there any merit to `rel="prev"` on the link?